### PR TITLE
PAB-3674 - Be more explicit about the dangers of CEP admin permissions

### PR DESCRIPTION
**Pull request by rthrippleton**

### CODE REVIEW

**Review by skom-sag — APPROVED** [review]

**Review by skom-sag — APPROVED** [review]

File: content/apama/analytics-introduction-bundle/developing-apps.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ your tenant must be subscribed to the Apama-ctrl microservice that supports EPL
 If you do not see the **EPL Apps** page in the Streaming Analytics application and you wish to use EPL apps, contact [product support](/welcome/contacting-support).
 {{< /c8y-admon-info >}}
 
+{{< c8y-admon-caution >}}
+An EPL app has the ability to make nearly arbitrary changes to the objects in a tenant, whether that's inventory, alarms or many other sorts of object. Because a user who has ADMIN permission for "CEP management" is able to create and active EPL apps, they also indirectly have nearly full control over the current tenant. Therefore you should be careful about which users on the tenant have this permission.
+{{< /c8y-admon-caution >}}
```
Comment (kailol-sag) [inline content/apama/analytics-introduction-bundle/developing-apps.md:22]: > they also indirectly have nearly full control over the current tenant

What does "they" refer to? It looks like this might refer to the plural "EPL apps". But I guess this is to refer to the "user" that you have at the start of the sentence. 

Maybe something like the following - our English teacher always told us not to start a sentence with "Because" ;-) 

A user who has ADMIN permission for "CEP management" is able to create and active EPL apps and can thus also indirectly have nearly full control over the current tenant.

**Comment by kailol-sag** [comment]
@rthrippleton Do we want to backport this to the version that is currently live? That is to 10.17, 10.16, and 10.15?

File: content/apama/analytics-introduction-bundle/developing-apps.md (hunk 1 above)
Comment (rthrippleton) [reply content/apama/analytics-introduction-bundle/developing-apps.md:22]: a4a23c493

Something isn't quite scanning though - `... and activate EPL apps and can thus also indirectly have nearly full control over ... ` reads like a long mix of words that's hard to parse. Would a comma after `apps` help, and maybe remove `indirectly` ? Hard to explain, it just doesn't feel right.

**Comment by rthrippleton** [comment]
> @rthrippleton Do we want to backport this to the version that is currently live? That is to 10.17, 10.16, and 10.15?

Sure. It's not hugely important, but go right ahead if it's just some easy label magic :-)

File: content/apama/analytics-introduction-bundle/developing-apps.md (hunk 1 above)
Comment (kailol-sag) [reply content/apama/analytics-introduction-bundle/developing-apps.md:22]: I think a comma would not be correct. How about the following:
"and thus indirectly also has almost full control over the current client."
And I don't mind if you want to remove "indirectly".

Comment (rthrippleton) [reply content/apama/analytics-introduction-bundle/developing-apps.md:22]: f7e114321

**Review by kailol-sag — APPROVED** [review]
